src/state/useAppState.js

```diff
@@ -0,0 +1,15 @@
+import React, { createContext, useContext } from 'react';
```

Comment:
@sbussard i'm totally cool if you want to move in this direction of building a `Provider` to manage our state. 
I think before we merge this, we will need to do the following: 
(1) Solidify the API 
(2) Solidify the pattern for actions and reducers 
(3) Have an example that everyone can work off of

Reply:
Another possibility since you want to remove `redux-thunk` is to look into `redux-saga`